pages/linux/medusa.md

```diff
@@ -0,0 +1,15 @@
+# Medusa
+
+> A speedy, parallel, and modular, login brute-forcer.
+
+- General:
```

Comment:
Please use verbs and full sentences to describe the functionality here.
I've left a suggestion but it probably does not fully describe what the command does so I would ask you to take that into consideration and rephrase it.
Same for the next examples. 
```suggestion
- Execute brute force using a list of usernames and passwords:
```